src/tests/ftest/util/server_utils.py

```diff
@@ -1038,6 +1038,23 @@ def stop_ranks(self, ranks, daos_log, force=False):
         # Update the expected status of the stopped/excluded ranks
         self.update_expected_states(ranks, ["stopped", "excluded"])
 
+    def kill(self):
+        """Forcibly terminate any server process running on hosts."""
+        regex = self.manager.job.command_regex
+        """Try to dump all server's ULTs stacks before kill."""
+        result = stop_processes(self._hosts, regex, dump_ult_stacks=True)
+        if 0 in result and len(result) == 1:
+            print(
+                "No remote {} server processes killed (none found), "
+                "done.".format(regex))
+        else:
+            print(
+                "***At least one remote {} server process needed to be killed! "
+                "Please investigate/report.***".format(regex))
+        # set stopped servers state to make teardown happy
+        self.server_managers[0].update_expected_states(
```

Comment:
(pylint-no-member) Instance of 'DaosServerManager' has no 'server_managers' member

Reply:
Ditto.